crates/proof-of-sql/src/base/scalar/scalar.rs

```diff
@@ -32,7 +32,8 @@ pub trait Scalar:
     + for<'a> core::convert::From<&'a i64> // Required for `Column` to implement `MultilinearExtension`
     + for<'a> core::convert::From<&'a i128> // Required for `Column` to implement `MultilinearExtension`
     + for<'a> core::convert::From<&'a u8> // Required for `Column` to implement `MultilinearExtension`
-    + for<'a> core::convert::From<&'a u64> // Required for `Column` to implement `MultilinearExtension`
+    + for<'a> core::convert::From<&'a u64> // Required for `Column` to implement `MultilinearExtension`\
+    + for<'a> core::convert::From<&'a [u8]>
```

Comment:
I want to avoid adding a `From` conversion where the conversion is potentially non-canonical.
Instead, in this case, add a new method to `ScalarExt`.